apps/condo/domains/organization/schema/OrganizationEmployee.js

```diff
@@ -81,8 +85,8 @@ const OrganizationEmployee = new GQLListSchema('OrganizationEmployee', {
             knexOptions: { isNotNullable: false },
             access: {
                 read: true,
-                create: access.userIsAdmin,
-                update: access.userIsAdmin,
+                create: userIsAdmin,
```

Comment:
Its OK. But it's better to use it like here: https://github.com/open-condo-software/condo/blob/master/apps/condo/domains/user/schema/User.js#L78